### DISCUSSION TITLE
new way to represent alphabets

### DISCUSSION THREAD

**Discussion by hurrican01** [khoj-ai/khoj#751] [Ideas]
When i was a childish child,i was lonely and seldom people will understand my thoughts,i like hide my thought and assume someone will break the code to sweat.That's one odd i touch code study.You many think it's amusing and 'what's the relationship between them.From my perspective,they still have something in common,they are all turn normal characters into other number objects.
To start with,i tried to number them like this.
A1B2C3D4 and so on
but it's complex and can make a complete sentense
just use simple number0,1 's combine 
thanks to permutation 2^4=16 2^5=32can cover all 26 alphabets. 32-26=6
ther's no capitals and lower case letters,just haveentire text start, word end,sentense end and entire text end,but it depends on individual.
a00101 b10110 c01001 d11100 e10110 f00111 g11010 h00100
i11011 j01101 k00010 l10100 m00110 n11111 o01000 p10001
q11110 r10101 s01111 t01010 u00011 v10000 w11000 x01011
y01100 z00001 
11101 10010 10111 00101 01011 11001
extra 6 freely

**Comment by debanjum** [comment]
Hi @hurrincane, is this idea somehow related to a feature request or issue you have for Khoj the product?

**Comment by hurrican01** [comment]
As humans all know,spacex's starship third and fourth emit has succesfully launched no matter in what perspectives,at once outerspace game made big business success,like many young people,i am crazy in playing games,finding rules in game and gain more victory as i can.Sadly i have to say that many outerspace game doesnt makes me satisfied.Next step you may say i am an idiot  in comment and say it as a joke to your friends,but trust me,i can prove that i am right.
Obviously if we want to make games more reality and excited, we have to make experiences for oneself more reality and have in touch with real world,that is one reason why players play games.Differnt from other types of games slightly increases,outerspace game still stains past paradigmi think maybe it's because people can't imagine what they dont experienced.Of course i cant,but i beieve in theory's power.Let's build a set which only contains players,known equipment and unknown equipment
Number1Law of conservation of energy,It states energy of a system may change forms, but it is neither created nor destroyed. One way to state this law is “you can’t get something for nothing”,which ctrlc and ctrlv in https://sciencenotes.org/laws-of-thermodynamics/,in that case we can't build a starship doesnt need energy to fly forever ,we have to gain energy from equipment.Exploring unknown zone is not only for psycho needs,but also for life needs.
Number2In an isolated system, there is no energy exchange between the system and the environment, and the system always changes spontaneously in the direction of increasing chaos, always increasing the entropy value of the whole system,life lives on negentropy.(schrodinger)To gain more energies,we must like angel investors invest every unpredicted condition.
Number3Occam's razor law，productors don't easily increase anything.
At last,nowadays ground floor game has matured.But its experiences has no examples for outerspace games instead of code,arithmetic power，example that ground floor game is like running in a wrinkled blanket,but outerspace is like a point in 0XYZ
